sirepo/raydata/analysis_driver/__init__.py

```diff
@@ -25,6 +29,9 @@ def __init__(self, catalog_name, uid, *args, **kwargs):
     def get_analysis_pdf_paths(self):
         return pkio.walk_tree(self.get_output_dir(), r".*\.pdf$")
 
+    def get_conda_env(self):
+        return ""
```

Comment:
This should be something like `raise NotImplementedError("children must implement this")`

Reply:
a4175ab0127a5b2207443ee18cab9d384c401eb9